sunpy/timeseries/sources/lyra.py

```diff
@@ -123,11 +120,7 @@ def peek(self, title=None, columns=None, names=3, **kwargs):
         axes = self.plot(columns=columns, names=names, **kwargs)
         if title is None:
             title = "LYRA ({0:{1}})".format(self.to_dataframe().index[0], TIME_FORMAT)
-        axes[0].set_title(title)
-        fig = axes[-1].get_figure()
-        fig.subplots_adjust(left=0.17, top=0.94, right=0.94, bottom=0.15)
-        fig = axes[0].get_figure()
-        fig.subplots_adjust(left=0.17, top=0.94, right=0.94, bottom=0.15)
+            axes[0].set_title(title)
```

Comment:
```suggestion
          axes[0].set_title(title)
```
Unless the title is already set?